index.html

```diff
@@ -0,0 +1,997 @@
+              &#x3C;img src=&#x22;./assets/home.svg&#x22; class=&#x22;badge-img-sm&#x22; /&#x3E;
+              &#x3C;span class=&#x22;badge-text-sm&#x22;&#x3E;42&#x3C;/span&#x3E;
+
+              &#x3C;div class=&#x22;badge&#x22;&#x3E;
+              &#x3C;img src=&#x22;./assets/home.svg&#x22; class=&#x22;badge-img-md&#x22; /&#x3E;
+              &#x3C;span class=&#x22;badge-text-md&#x22;&#x3E;42&#x3C;/span&#x3E;
+            &#x3C;/div&#x3E;
+
+            &#x3C;div class=&#x22;badge&#x22;&#x3E;
+              &#x3C;img src=&#x22;./assets/home.svg&#x22; class=&#x22;badge-img-lg&#x22; /&#x3E;
+              &#x3C;span class=&#x22;badge-text-lg&#x22;&#x3E;42&#x3C;/span&#x3E;
+            &#x3C;/div&#x3E;
+            </code>
+          </pre>
+        </div>
+        <h4>Badge on avatar</h4>
+          <div class="flex row-reverse align-items-center justify-content-space-between">
+            <div class="badge">
+            <img
+            src="https://cdn.discordapp.com/avatars/778699572787675136/bfd4785c0eb8a2be4eae7cc2fec27d34.png?size=256&quot"
+            class="avatar-sm"
+            />
+            <span class="badge-avatar-text-sm ">41</span>
+            </div>
+        <div class="badge">
+          <img
+            src="https://cdn.discordapp.com/avatars/778699572787675136/bfd4785c0eb8a2be4eae7cc2fec27d34.png?size=256&quot"
+            class="avatar-md"
+          />
+          <span class="badge-avatar-text-md">41</span>
+        </div>
+        <div class="badge">
+          <img
+            src="https://cdn.discordapp.com/avatars/778699572787675136/bfd4785c0eb8a2be4eae7cc2fec27d34.png?size=256&quot"
+            class="avatar-lg"
+          />
+          <span class="badge-avatar-text-lg">41</span>
+        </div>
+      </div>
+      <div class="code">
+        <pre >
+          <code class="language-html">
+            &#x3C;div class=&#x22;badge&#x22;&#x3E;
+            &#x3C;img src="..." class=&#x22;avatar-lg&#x22;/&#x3E;
+            &#x3C;span class=&#x22;badge-avatar-text-lg&#x22;&#x3E;41&#x3C;/span&#x3E;
+          &#x3C;/div&#x3E;
+
+          &#x3C;div class=&#x22;badge&#x22;&#x3E;
+            &#x3C;img src="..." class=&#x22;avatar-md&#x22;/&#x3E;
+            &#x3C;span class=&#x22;badge-avatar-text-md&#x22;&#x3E;41&#x3C;/span&#x3E;
+          &#x3C;/div&#x3E;
+
+        &#x3C;div class=&#x22;badge&#x22;&#x3E;
+          &#x3C;img src="..." class=&#x22;avatar-sm&#x22;/&#x3E;
+          &#x3C;span class=&#x22;badge-avatar-text-sm &#x22;&#x3E;41&#x3C;/span&#x3E;
+        &#x3C;/div&#x3E;
+          </code>
+        </pre>
+      </div>
+      </div>
+      <div class="pd-1 mg-bottom-1" id="button">
+        <h3>Button</h3>
+        <div class="row-responsive" >
+          <button class="btn-primary-lg">Primary Button</button>
+          <button class="btn-primary-md mg-top-half mg-bottom-half">Primary Button</button>
+          <button class="btn-primary-sm">Primary Button</button>
```

Comment:
Padding seems a little less for `btn-primary-sm`. Maybe increase it or make it similar to other buttons.